node/src/main/scala/org/bitcoins/node/SpvNode.scala

```diff
@@ -85,9 +85,15 @@ case class SpvNode(
     }
   }
 
-  override def rescan(
-      scriptPubKeysToWatch: Vector[ScriptPubKey],
-      startOpt: Option[BlockStamp],
-      endOpt: Option[BlockStamp]): Future[Unit] =
-    Future.failed(new RuntimeException("Rescan not implemented"))
+  /** Gets the number of compact filters in the database */
+  override def getFilterCount: Future[Int] = Future.successful(0)
```

Comment:
Can we just make these `Future.failed()` for now so we get errors rather than `0`?